src/features/feedback/components/FeedbackComment.tsx

```diff
@@ -16,7 +17,10 @@ interface FeedbackCommentProps {
 export const FeedbackComment = ({ post }: FeedbackCommentProps) => {
   const prettyDate = useMemo(() => {
     if (!post) return ''
-    return format(new Date(post.createdAt), 'dd MMM yyyy, hh:mmaaa')
+    return format(
+      utcToZonedTime(new Date(post.createdAt), 'Asia/Singapore'),
```

Comment:
May not necessarily always be UTC. Since it was using new Date() invocation, should we use the formatInTimeZone fn instead?